tools/lsp/common.rs

```diff
@@ -28,8 +28,19 @@ pub fn filter_ignore_nodes_in_element(
     element: &Element,
 ) -> impl Iterator<Item = &syntax_nodes::Element> {
     element.node.iter().filter(move |e| {
-        !e.children().any(|n| {
-            n.kind() == SyntaxKind::Comment && format!("{}", n.text()).contains(NODE_IGNORE_COMMENT)
+        !e.children_with_tokens().any(|nt| {
+            nt.as_node()
+                .map(|n| {
+                    n.kind() == SyntaxKind::Comment
+                        && format!("{}", n.text()).contains(NODE_IGNORE_COMMENT)
```

Comment:
Does that work?
```suggestion
                        && n.text().contains(NODE_IGNORE_COMMENT)
```

No need to format it.

n.text().to_string()  should also work.

Yes i know, this was there before

Reply:
If one can avoid allocating, yes. Other it's just not good practice to do `format!("{}"`
There is even a clippy lint for it: https://rust-lang.github.io/rust-clippy/stable/index.html#/useless_format